src/plugins/shared_ux/public/components/toolbar/solution_toolbar/quick_group_buttons/quick_group.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import {
+  EuiButtonGroup,
+  htmlIdGenerator,
+  EuiButtonGroupOptionProps,
+  useEuiTheme,
+  IconType,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+
+const strings = {
+  getAriaButtonLabel: (createType: string) =>
+    i18n.translate('sharedUX.solutionToolbar.quickButton.ariaButtonLabel', {
+      defaultMessage: `Create new {createType}`,
+      values: {
+        createType,
+      },
+    }),
+  getLegend: () =>
+    i18n.translate('sharedUX.solutionToolbar.quickButton.legendLabel', {
+      defaultMessage: 'Quick create',
+    }),
+};
+
+export interface QuickButtonProps {
+  createType: string;
+  onClick: () => void;
+  iconType: IconType;
+}
+
+export interface Props {
+  buttons: QuickButtonProps[];
+}
+
+type Option = EuiButtonGroupOptionProps & Omit<QuickButtonProps, 'createType'>;
+
+export const QuickButtonGroup = ({ buttons }: Props) => {
+  const { euiTheme } = useEuiTheme();
+
+  const buttonGroupOptions: Option[] = buttons.map((button: QuickButtonProps, index) => {
+    const { createType: label, ...rest } = button;
+    const title = strings.getAriaButtonLabel(label);
+
+    return {
+      ...rest,
+      'aria-label': title,
+      id: `${htmlIdGenerator()()}${index}`,
+      label,
+      title,
+      className: `quickButtonGroup__button`,
```

Comment:
We shouldn't be using classnames, but rather pure emotion styles... or am I missing something here?

Reply:
[74c44b5](https://github.com/elastic/kibana/pull/126297/commits/74c44b5b073476a46ce2028fb1a21e4fef80e272) for changes thanks!